report/thermodesorption-spectra/ogorodnikova-tds.ipynb

```diff
@@ -0,0 +1,299 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Thermo-desorption spectrum\n",
+    "\n",
+    "This validation case is a thermo-desorption spectrum measurement perfomed by Ogorodnikova et al. {cite}`ogorodnikova_deuterium_2003`.\n",
+    "\n",
+    "Deuterium ions at 200 eV were implanted in a 0.5 mm thick sample of high purity tungsten foil (PCW).\n",
+    "\n",
+    "The ion beam was turned on for 400 s which corresponds to a fluence of $1.0 \\times 10^{22} \\ \\mathrm{D \\ m^{-2}}$\n",
+    "\n",
+    "The trapping parameters are taken from Table 1 for both the intrinsic traps and the extrinsic, ion-induced traps."
```

Comment:
"taken from Table 1", which Table 1? is it from Ogorodnikova et al? I don't think so.

You don't need to give a reference for the parameters that are not experimental parameters.